ext/gmp/php_gmp_int.h

```diff
@@ -28,7 +28,7 @@ static inline gmp_object *php_gmp_object_from_zend_object(zend_object *zobj) {
 PHP_GMP_API zend_class_entry *php_gmp_class_entry(void);
 
 /* GMP and MPIR use different datatypes on different platforms */
-#ifdef PHP_WIN32
+#ifdef _WIN64
```

Comment:
@nielsdos you are the compiler expect, do you know why this change is needed and if it is correct?

Reply:
Previously, both 32-bit and 64-bit Windows used zend_(u)long for gmp_(u)long.
With this patch, 32-bit Windows uses (u)long and 64-bit Windows uses zend_(u)long.
Note that zend_long is typedeffed as int32_t on 32-bit (so it is indirectly typedeffed as an int). On 32-bit Windows long is 32-bit as well, but still its own type. Strictly speaking they are different types (int vs long) and I think the compiler emitting a warning is just overly cautious here.
I didn't test it, but the change _looks_ correct.